.github/workflows/build.yml

```diff
@@ -31,3 +31,21 @@ jobs:
           ANACONDA_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
         run: |
           ./build.sh --include-user-channel --token "$ANACONDA_TOKEN" --push
+
+  build-macos-conda-packages:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: gadgetron-conda-recipes
+          environment-file: environment.yml
+          python-version: 3.9
+          auto-activate-base: false
+      - name: Build macos conda packages
+        shell: bash -l {0}
+        if: ${{ github.event_name == 'pull_request' }}
```

Comment:
We need the push equivalent too. That is the run that runs when we merge and that one should push to anaconda.org (see how it is done for Linux).

Reply:
Yes - will do that eventually.

Since the actions on 'push' seems to create packages for conda-forge, I wanted to make sure we were all happy with how the macOS runners and the Mac builds were behaving, before we complete that workflow.